.circleci/config.yml

```diff
@@ -8,8 +8,11 @@ jobs:
     - image: circleci/python:3.7
     steps:
       - checkout
-      - run: sudo pip3 install tox -c constraints.txt
-      - run: tox -e lint
+      - run:
+          name: run linting
+          command: |
+            sudo pip3 install pre-commit -c constraints.txt
+            pre-commit run --all-files
```

Comment:
otherwise known as `make lint`. We also block PRs by linting in the daint setup step, should we remove it from there?

Reply:
If you or someone else still runs `make lint` then we should move it to the top level - I haven't run it this way in a very long time, since it happens automatically on commit.